bin/sce_to_anndata.R

```diff
@@ -56,6 +62,17 @@ if (!(stringr::str_ends(opt$output_rna_h5, ".hdf5|.h5|.h5ad"))) {
   stop("output rna file name must end in .hdf5, .h5, or .h5ad")
 }
 
+# Merged object function  ------------------------------------------------------
+
+# this function updates merged object formatting for anndata export
+format_merged_sce <- function(sce) {
+  # paste X to reduced dim names if present
+  if (!is.null(reducedDimNames(sce))) {
+    reducedDimNames(sce) <- glue::glue("X_{reducedDimNames(sce)}")
+  }
```

Comment:
By my quick test, this is not `NULL` if there are no reducedDims; it is an empty vector. Since `glue` with an empty vector will return an empty vector, we don't need the `if` 
```suggestion
  reducedDimNames(sce) <- glue::glue("X_{reducedDimNames(sce)}")
```

I realize that this is in the `format_czi` code as well, but we should update that too!